algebra/u256/src/additive.rs

```diff
@@ -158,12 +156,10 @@ mod tests {
         assert_eq!(a, e);
     }
 
-    #[quickcheck]
-    fn commutative_add(a: U256, b: U256) -> bool {
-        let mut l = a.clone();
-        let mut r = b.clone();
-        l += &b;
-        r += &a;
-        l == r
-    }
+    proptest!(
```

Comment:
Curly brackets? (also in following)

Reply:
cargo fmt ignores expressions in `{}`. I checked that this is happening in the libra codebase as well.

https://github.com/rust-lang/rustfmt/issues/2867